integtest/spec/all_books_sources_spec.rb

```diff
@@ -71,7 +71,7 @@ def self.init_docs_repo(src)
           extra_branches.each { |b| repo.switch_to_new_branch b }
           docs_repo = init_docs_repo src
           book = src.book 'Test'
-          book.branches << extra_branches
+          book.branches += extra_branches
```

Comment:
It looks like the old way was tolerant of the "funny" array this used to build: `[master, [5.5, 6.3, 7.2]]` but the new way isn't. So it caught a bug!